pkgs/development/python-modules/connexion/default.nix

```diff
@@ -0,0 +1,82 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, isPy3k
+, glibcLocales
+, lib
+, pythonOlder
+
+, aiohttp
+, aiohttp-swagger
+, aiohttp-jinja2
+, clickclick
+, decorator
+, flake8
+, flask
+, gevent
+, inflection
+, jsonschema
+, mock
+, openapi-spec-validator
+, pathlib
+, pytest
+, pytest-aiohttp
+, pytestcov
+, pyyaml
+, requests
+, six
+, swagger-ui-bundle
+, testfixtures
+, typing
+, ujson }:
+
+buildPythonPackage rec {
+  pname = "connexion";
+  version = "2.2.0";
+
+  # we're fetching from GitHub because tests weren't distributed on PyPi
+  src = fetchFromGitHub {
+    owner = "zalando";
+    repo = pname;
+    rev = version;
+    sha256 = "1psbql502xa7g8q4m7mpqpdicyb2kffgq9wvrq9jr7ld22k2m282";
+  };
+
+  buildInputs = lib.optional (pythonOlder "3.7") glibcLocales;
```

Comment:
Was this was only needed for the tests?

If so you can put it in `checkInputs`.

Reply:
Also note that `checkInputs` are `nativeBuildInputs` for now.